commons/com.b2international.index/src/com/b2international/index/es/admin/EsIndexAdmin.java

```diff
@@ -391,7 +412,9 @@ private void waitForYellowHealth(String... indices) {
 						if (!keywordMapping.index()) {
 							prop.put("index", false);
 						}
-						prop.put("doc_values", keywordMapping.index());
+						if (!keywordMapping.index()) {
+							prop.put("doc_values", keywordMapping.index());
+						}
```

Comment:
The two `if (!keywordMapping.index())` tests could be merged together; the property value could also be converted to a `false` literal for consistency.